presto-hive/src/test/java/com/facebook/presto/hive/parquet/write/TestDataWritableWriter.java

```diff
@@ -368,7 +371,13 @@ private void writePrimitive(final Object value, final PrimitiveObjectInspector i
                 break;
             case TIMESTAMP:
                 Timestamp ts = ((TimestampObjectInspector) inspector).getPrimitiveJavaObject(value);
-                recordConsumer.addBinary(NanoTimeUtils.getNanoTime(ts, false).toBinary());
+                final NanoTime nanoTime = NanoTimeUtils.getNanoTime(ts, false);
```

Comment:
We don't use `final` for local variables, please remove.

Reply:
I need to come up with some checkstyle rule, intellij by default seem to always put final when I instruct it to extract a local variable. Done.